packages/flutter/lib/src/material/feedback.dart

```diff
@@ -0,0 +1,73 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+/// Provides platform-specific acoustic and/or haptic feedback for certain
```

Comment:
These docs need to be more elaborate.

Things to add at each place there are docs in this file (it's perfectly ok to have duplication in the docs):
* What does this do on Android?
* What does this do on iOS?
* How do you handle things like `onPressed: enabled ? callback : null`?
* Sample code? (with the `## Sample code` pattern, see other files)